generative_ai/batch_predict/noxfile_config.py

```diff
@@ -22,7 +22,7 @@
 
 TEST_CONFIG_OVERRIDE = {
     # You can opt out from the test for specific Python versions.
-    "ignored_versions": ["2.7", "3.7", "3.8", "3.10", "3.11", "3.13"],
+    "ignored_versions": ["2.7", "3.7", "3.8", "3.10", "3.13"],
```

Comment:
@msampathkumar I thought we were only going to test on 3.9 and 3.12

Reply:
that was modified for local testing. I'll update to add 3.11 back